examples/tutorial/05_Interactive_Pipelines.ipynb

```diff
@@ -311,6 +311,54 @@
     "For readability, seven columns were chosen before displaying the `DataFrame`. Have a look at `df.columns` and pick a different set of columns for display."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Functions as inputs\n",
+    "\n",
+    "Quite often the data structure you want to explore in a pipeline, e.g. a Pandas DataFrame, may itself be the outcome of another pipeline. It may for instance be extracted and transformed from a database or an API call, or the output of some simulation or pre-processing. With `hvplot.bind` you can bind an input function that returns the data structure you want to explore, to widgets that when updated will re-execute that function. \n",
```

Comment:
```suggestion
    "Quite often the data structure you want to explore in a pipeline, may itself be the outcome of another pipeline. It may for instance be a Pandas Dataframe created by extracting and transforming the output of a database or an API call, or it could be the dynamic output of some simulation or pre-processing. With `hvplot.bind` you can start with an arbitrary custom function that returns the data structure you want to explore and then bind it to widgets that will call that function for updated output whenever the widgets change.\n",
```